src/config_cmd.go

```diff
@@ -142,15 +142,47 @@ func (c *ConfigCmd) Execute(args []string) error {
 	// or maybe a custom stale checker is better than just using file mtimes for
 	// all the files (maybe just use setup.py as a prereq? but then how will we
 	// update SourceUnit.Files list? SourceUnit.Globs could help here...)
+	//
+	// note(samertm): going with a custom checker because we need to create a
```

Comment:
By "custom checker", I mean we're relying on srclib & the scm for whether we should rebuild a unit, not globs and the makefile. This comment is confusing, I'll remove it.